.settings/module_db.json

```diff
@@ -3568,4 +3568,4 @@
       "requirements": []
     }
   ]
-}
```

Comment:
@Marvmann Is "qrisp" now a new dependency? If yes the `module_db` should be updated.

Reply:
yes, and module_db has been adjusted!